<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>动态地图管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>起点站：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="startStation"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>终点站：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="terminateStation"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>当前站：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="currentStation"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>下一站：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="nextStation"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBoxArriveLeft">
           <property name="minimumSize">
            <size>
             <width>67</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>67</width>
             <height>70</height>
            </size>
           </property>
           <property name="title">
            <string>到离站</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QRadioButton" name="leaveTrigger">
              <property name="text">
               <string>离站</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="arriveTrigger">
              <property name="text">
               <string>到站</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton">
              <property name="text">
               <string>RadioButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxOpenDoor">
           <property name="minimumSize">
            <size>
             <width>91</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>91</width>
             <height>70</height>
            </size>
           </property>
           <property name="title">
            <string>开门侧</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QCheckBox" name="rightDoorOpen">
              <property name="text">
               <string>右侧开门</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="leftDoorOpen">
              <property name="text">
               <string>左侧开门</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxTC">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="title">
            <string>车头激活</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QRadioButton" name="tc1Active">
              <property name="text">
               <string>TC1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="tc2Active">
              <property name="text">
               <string>TC2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_2">
              <property name="text">
               <string>RadioButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBoxVolume">
           <property name="title">
            <string>音量</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QSlider" name="horizontalSliderVolume">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxVolume">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxLuminance">
           <property name="title">
            <string>亮度</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QSlider" name="horizontalSliderLuminance">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxLuminance">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>紧急广播:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxEA"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxSkip">
       <property name="title">
        <string>越站设置</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>当前交路所有车站</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidgetAll"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButtonAdd">
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDel">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonClr">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>设置了跳站的车站</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidgetSkip"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditCommand">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="canclePushButton">
       <property name="text">
        <string>取消（&amp;C)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OK">
       <property name="text">
        <string>确定（&amp;O）</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderVolume</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxVolume</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderLuminance</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxLuminance</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxVolume</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderVolume</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxLuminance</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderLuminance</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
